h5c/vic/src/vic-webapp/src/app/vch-view/vch-view.template.html

```diff
@@ -50,6 +50,7 @@
   <clr-dg-row *ngFor="let item of vchs">
     <clr-dg-action-overflow class="{{item.name}}">
       <button class="action-item {{item.name}}" (click)="deleteVch(item)">Delete</button>
+      <button class="action-item cert {{item.name}}" (click)="vchCert(item)">Download VCH certificate</button>
```

Comment:
Function `vchCert` does not exist on the VicVchViewComponent and it breaks the application.